source/install/desktop-app-install.rst

```diff
@@ -24,7 +24,7 @@ You can `download the desktop app directly from our Downloads page <https://matt
 
   .. tab:: Ubuntu/Debian
 
-    A ``.deb`` package is available for Debian 9 and for Ubuntu releases 18.04 LTS or later. Automatic app updates aren’t supported. You must update your app manually.
+    An official ``.deb`` package and APT repository is available for Debian 9 and for Ubuntu releases 18.04 LTS or later. Automatic app updates are supported and enabled. When a new version of the desktop app is released, your app updates automatically.
```

Comment:
Sorry @cwarnermm I misspoke before. The `.deb` that we provide from GitHub is technically unofficial, but the APT repository is official.

Reply:
Thanks! I've noted that the ``.deb`` package is in beta and the APT repo is official.